### ISSUE TITLE
Aether Lance does not increase the scope of the Degen Aura ability

### ISSUE THREAD

**Issue by Jijonia** [ValveSoftware/Dota2-Gameplay#2117]
### Ability name

Degen Aura

### Description

Aether Lance does not increase the scope of the Degen Aura ability, when hovering over the ability, the radius is only visually detected.
![image](https://user-images.githubusercontent.com/107216529/184896662-d37a8856-b9b6-4ea4-af2d-977b466af30c.png)
![image](https://user-images.githubusercontent.com/107216529/184896699-439ee3b3-9478-477b-92a0-792563cd0808.png)
the screenshots show that the hedgehog has no status effect from the aura

### Example Match ID (and possibly Timestamp)

tested in a sample of the hero

**Comment by TheUnholyWrath** [comment]
Duplicate of #1712

**Comment by smmarius** [comment]
I don't think it should work on passive skills anyway. Otherwise it should work on items like Vladimirs as well.
